src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -180,6 +184,22 @@ class RootCoordinator: RequiresAppDependencies {
 		)
 	}
 	
+	func showHealthCertifiedPersonFromNotification(for healthCertifiedPerson: HealthCertifiedPerson) {
+		
+		guard let healthCertificateNavigationController = healthCertificatesCoordinator?.viewController,
+			  let index = tabBarController.viewControllers?.firstIndex(of: healthCertificateNavigationController) else {
+			Log.warning("Could not show Person because the corresponding navigation controller. can't be found")
+			return
+		}
+
+		// Close all modal screens that would prevent showing the certificate.
+		tabBarController.dismiss(animated: false)
```

Comment:
Should all modal screens be closed? There could be an onboarding screen with some legal text which needs to be approved. How are we handling this cases?

Reply:
This was already implemented, i think it was misread as new lines during merge conflicts